test/Api/CatalogApiTest.php

```diff
@@ -0,0 +1,640 @@
+        "tax_ids" => array("#SalesTax"),
+        "variations" => array(
+          array(
+            "type" => "ITEM_VARIATION",
+            "id" => "#SmallCoffee",
+            "item_variation_data" => array(
+              "name" => "Small",
+              "item_id" => "#Coffee",
+              "pricing_type" => "FIXED_PRICING",
+              "price_money" => array(
+                "amount" => 195.0,
+                "currency" => "USD"
+              )
+            )
+          ),
+          array(
+            "type" => "ITEM_VARIATION",
+            "id" => "#LargeCoffee",
+            "item_variation_data" => array(
+              "name" => "Large",
+              "item_id" => "#Coffee",
+              "pricing_type" => "FIXED_PRICING",
+              "price_money" => array(
+                "amount" => 255.0,
+                "currency" => "USD"
+              )
+            )
+          )
+        )
+      )
+    );
+    private $tea = array(
+      "type" => "ITEM",
+      "id" => "#Tea",
+      "item_data" => array(
+        "name" => "Tea",
+        "description" => "Hot leaf juice",
+        "abbreviation" => "Te",
+        "category_id" => "#Beverages",
+        "modifier_list_info" => array(array("modifier_list_id" => "#Milks")),
+        "tax_ids" => array("#SalesTax"),
+        "variations" => array(
+          array(
+            "type" => "ITEM_VARIATION",
+            "id" => "#SmallTea",
+            "item_variation_data" => array(
+              "name" => "Small",
+              "item_id" => "#Tea",
+              "pricing_type" => "FIXED_PRICING",
+              "price_money" => array(
+                "amount" => 150.0,
+                "currency" => "USD"
+              )
+            )
+          ),
+          array(
+            "type" => "ITEM_VARIATION",
+            "id" => "#LargeTea",
+            "item_variation_data" => array(
+              "name" => "Large",
+              "item_id" => "#Tea",
+              "pricing_type" => "FIXED_PRICING",
+              "price_money" => array(
+                "amount" => 200.0,
+                "currency" => "USD"
+              )
+            )
+          )
+        )
+      )
+    );
+    private $sales_tax = array(
+      "type" => "TAX",
+      "id" => '#SalesTax',
+      "present_at_all_locations" => true,
+      "tax_data" => array(
+        "name" => "Sales Tax",
+        "calculation_phase" => "TAX_SUBTOTAL_PHASE",
+        "inclusion_type" => "ADDITIVE",
+        "percentage" => '5.0',
+        "applies_to_custom_amounts" => true,
+        "enabled" => true
+      )
+    );
+
+
+
+    /**
+     * Setup before running each test case
+     */
+    public static function setUpBeforeClass() {
+      self::$api = new \SquareConnect\Api\CatalogApi();
+      self::$test_accounts = new \SquareConnect\TestAccounts();
+      // Configure OAuth2 access token for authorization: oauth2
+      $account = self::$test_accounts->{'US-Prod'};
```

Comment:
What does this `->{ ... }` syntax mean? I don't think I've encountered this before.